src/common/translation/translation-set.ts

```diff
@@ -109,6 +109,10 @@ export interface TranslationSet {
     appearanceSettingsResetWarningMessage: string;
     appearanceSettingsAllowTransparentBackground: string;
     appearanceSettingsFontFamily: string;
+    appearanceSettingsUserInputRadius: string;
+    appearanceSettingsUserInputBottomMargin: string;
+    appearanceSettingsSearchResultsRadius: string;
+    appearanceSettingsScrollbarRadius: string;
```

Comment:
Can you rename these variables also so that they contain "BorderRadius" instead of just "Radius"?